src/core/namespace.c

```diff
@@ -1139,7 +1140,11 @@ static int mount_bind_dev(const MountEntry *m) {
         if (r > 0) /* make this a NOP if /dev is already a mount point */
                 return 0;
 
-        return mount_nofollow_verbose(LOG_DEBUG, "/dev", mount_entry_path(m), NULL, MS_BIND|MS_REC, NULL);
+        r = mount_nofollow_verbose(LOG_DEBUG, "/dev", mount_entry_path(m), NULL, MS_BIND|MS_REC, NULL);
+        if (r < 0)
+                return log_debug_errno(r, "Failed to mount /dev to %s: %m", mount_entry_path(m));
```

Comment:
hmm? you already using the verbose flavour of mount_nofollow(), hence no need to log here manually